nni/compression/pytorch/speedup/infer_shape.py

```diff
@@ -891,23 +891,18 @@ def convert_to_coarse_mask(mask, dim=0):
         sum_idx = (1, 2, 3) if dim == 0 else (0, 2, 3)
         index = torch.nonzero(weight_mask.abs().sum(
             sum_idx) != 0, as_tuple=True)[0]
-        if len(index) == weight_mask.shape[dim]:  # full mask
-            index = None
```

Comment:
what issue does `index = None` cause, are there other situations that can cause `index == None`?